bot/config.py

```diff
@@ -17,13 +17,20 @@
 use_chatgpt_api = config_yaml.get("use_chatgpt_api", True)
 allowed_telegram_usernames = config_yaml["allowed_telegram_usernames"]
 new_dialog_timeout = config_yaml["new_dialog_timeout"]
+mongo_host = config_env.get("MONGODB_HOST", "mongo")
 enable_message_streaming = config_yaml.get("enable_message_streaming", True)
-mongodb_uri = f"mongodb://mongo:{config_env['MONGODB_PORT']}"
+mongodb_uri = f"mongodb://{mongo_host}:{config_env['MONGODB_PORT']}"
 
 # chat_modes
-with open(config_dir / "chat_modes.yml", 'r') as f:
+with open(config_dir / "chat_modes.yml", 'r', encoding="utf8") as f:
```

Comment:
without define the encoding it faileds load chat_modes.yml on windows
because windows default encoding is unicode-16